src/components/Login/Login.jsx

```diff
@@ -1,7 +1,94 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import './Login.scss';
 
 function Login() {
-  return <h1>Login</h1>;
+  const navigate = useNavigate();
+  const [inputValue, setInputValue] = useState({
+    username: '',
+    password: '',
+  });
+  const handleChangeInput = e => {
+    const { name, value } = e.target;
+    setInputValue({ ...inputValue, [name]: value });
+  };
```

Comment:
```suggestion
  const handleChangeInput = e => {
    const { name, value } = e.target;
    setInputValue(prev => ({ ...prev, [name]: value }));
  };
```
함수형 업데이트 방식으로 구현하는 것이 조금 더 효율적인 방식입니다!

Reply:
수정 했습니다!